bulla-contracts/src/mappings/BullaFactoring.ts

```diff
@@ -30,15 +30,19 @@ export function handleInvoiceFunded(event: InvoiceFunded): void {
   InvoiceFundedEvent.fundedAmount = ev.fundedAmount;
   InvoiceFundedEvent.originalCreditor = ev.originalCreditor;
   const original_creditor = getOrCreateUser(ev.originalCreditor);
+  // Update the price history
   const price_per_share = getOrCreatePricePerShare(event);
```

Comment:
Not used?

Reply:
don't we use that for the pricing of BFT? we need the array of historical prices

Reply:
we save it at the bottom of each function to have it available

Reply:
Doesn't this change replace that mecanism?